app/code/Magento/Catalog/view/frontend/templates/product/image_with_borders.phtml

```diff
@@ -4,16 +4,20 @@
  * See COPYING.txt for license details.
  */
 ?>
-<?php /** @var $block \Magento\Catalog\Block\Product\Image */ ?>
+<?php
+/** @var $block \Magento\Catalog\Block\Product\Image */
+/** @var $escaper \Magento\Framework\Escaper */
+?>
 
 <span class="product-image-container"
-      style="width:<?= $block->escapeHtmlAttr($block->getWidth()) ?>px;">
+      style="width:<?= $escaper->escapeHtmlAttr($block->getWidth()) ?>px;">
     <span class="product-image-wrapper"
           style="padding-bottom: <?= ($block->getRatio() * 100) ?>%;">
-        <img class="<?= $block->escapeHtmlAttr($block->getClass()) ?>"
-            <?= $block->escapeHtmlAttr($block->getCustomAttributes()) ?>
-            src="<?= $block->escapeUrl($block->getImageUrl()) ?>"
-            max-width="<?= $block->escapeHtmlAttr($block->getWidth()) ?>"
-            max-height="<?= $block->escapeHtmlAttr($block->getHeight()) ?>"
-            alt="<?= /* @noEscape */ $block->stripTags($block->getLabel(), null, true) ?>"/></span>
+        <img class="<?= $escaper->escapeHtmlAttr($block->getClass()) ?>"
+             <?= $escaper->escapeHtmlAttr($block->getCustomAttributes()) ?>
+             src="<?= $escaper->escapeUrl($block->getImageUrl()) ?>"
+             loading="lazy"
+             width="<?= $escaper->escapeHtmlAttr($block->getWidth()) ?>"
```

Comment:
Could you explain why you changed max-width and max-height to just width and height? It looks not related to this PR.

As max width and max height are not valid attributes - seems like we could just remove them

Reply:
The width and height attributes are needed to preserve the aspect ratio of the image when it is not loaded yet.

Reply:
Ok, no problem, but need just test it carefully